examples/mt-pytorch/task.py

```diff
@@ -12,7 +12,7 @@
 
 
 warnings.filterwarnings("ignore", category=UserWarning)
-DEVICE = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+DEVICE = torch.device("cpu")
```

Comment:
Is this necessary?

Reply:
On my machine the GPU didn't work, but I didn't look too much into it, I will check this

Reply:
This is fixed now